Library/Formula/gauche.rb

```diff
@@ -11,11 +11,23 @@ class Gauche < Formula
     sha1 "087b1f85a18485fe2226892d9b96066a027de606" => :lion
   end
 
+  option "with-slib", "Configure Gauche's slib module"
```

Comment:
We can't, really. We need to not rely on a scheme implementation that does that.